pdf_rendering_service/documents/services/render.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from django.conf import settings
+from pdf2image import convert_from_bytes
+from pdf2image.exceptions import (
+    PDFInfoNotInstalledError,
+    PDFPageCountError,
+    PDFSyntaxError,
+)
+from PIL import Image
+from resizeimage import resizeimage
+
+from pdf_rendering_service.documents.models import Document
+from pdf_rendering_service.documents.services.document_store import (
+    get_document_file,
+    store_document_page,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def render_document_into_png_files(document: Document):
+    try:
+        images = convert_from_bytes(get_document_file(document).read())
+    except (PDFInfoNotInstalledError, PDFPageCountError, PDFSyntaxError) as e:
+        logger.error(
+            "PDF document id: %s could not be processed because of err: %s",
+            document.id,
+            e.__class__.__name__,
+        )
+        return
+
+    for page_num, image in enumerate(images):
+        image = normalize_document_page_image(image)
+        store_document_page(document.id, page_num + 1, image)
```

Comment:
just FYI: `enumerate` has `start` argument:
```suggestion
    for page_num, image in enumerate(images, start=1):
        image = normalize_document_page_image(image)
        store_document_page(document.id, page_num, image)
```
and just a philosophical question: what do you think about pages numbers starting from 0?

Reply:
> what do you think about pages numbers starting from 0?

In most cases it makes more sense to me. But for this case, where requesting user may not understand that pages are numbered from 0, it makes more sense to start numbering from 1. The file numbers are the same as the url

Reply:
Thanks for the response :)